analyzers/tests/SonarAnalyzer.UnitTest/Rules/PublicMethodWithMultidimensionalArrayTest.cs

```diff
@@ -42,6 +42,16 @@ public class PublicMethodWithMultidimensionalArrayTest
                 .WithOptions(ParseOptionsHelper.FromCSharp11)
                 .Verify();
 
+#endif
+
+#if NET48_OR_GREATER
```

Comment:
As discussed offline we could try to change the `MetadataReferenceFactory` and use the .NET 8 system assemblies for these tests.

Reply:
Answered [here](https://github.com/SonarSource/sonar-dotnet/pull/8082#discussion_r1341079677)